hmcts/pre-api
# S28 3127: Fix Create Endpoint + Locator Endpoint Without .ism 500 Error

### JIRA ticket(s)
- S28-3127


### Change description
- Fix 500 error when you call `PUT /media-service/streaming-locator/live-event/{captureSessionId}` without starting the recording (no .ism file found). 
  - Prevent creation of streaming endpoint + locator when recording has not started (so there is no .ism file)




> [!IMPORTANT]
> This PR requires manual testing by QA. Please notify the QA team @hmcts/pre-rec-evidence-qa.

**QA instructions**
- Create a capture session. Start the capture session. Do not start the recording on CVP. Call this endpoint. See 404 error- see endpoint + locator not created in AMS/MK
- Create a capture session. Start the capture session. Start CVP recording. Call this endpoint. See 200, no error. See endpoint + locator created in AMS/MK
